genestack_client/genestack_connection.py

```diff
@@ -240,7 +240,7 @@ def perform_request(self, path, data='', follow=True, headers=None):
             _headers.update(headers)
         try:
             response = self.session.post(self.server_url + path, data=data, headers=_headers,
-                                         allow_redirects=follow)
+                                         allow_redirects=follow, timeout=os.environ.get("CLIENT_TIMEOUT", None))
```

Comment:
This variable should be more Genestack specific: `GENESTACK_CLIENT_TIMEOUT`

Is `os.environ.get("CLIENT_TIMEOUT", None)` an integer?

Reply:
In case this question is not sarcastic or something, it's `str` or `None`.

Reply:
Once again: too long lines
![bild](https://user-images.githubusercontent.com/5174481/58566918-1c898780-823a-11e9-9099-c8d656424779.png)